### PR TITLE
[WIP] Deal with multiple manifests/lockfiles

### PR DESCRIPTION
**Pull request by paulortman**
Addresses #6

### CODE REVIEW

File: tests/collector/basic_pip/expected_output_data.json [hunk 1]
```diff
@@ -7,181 +7,202 @@
...
-            ], 
-            "constraint": "17.5.0", 
+            ],
+            "constraint": "17.5.0",
+            "source": "pypi"
+          }
+        }
+      }
+    },
+    "sub/requirements.txt": {
```
Comment (paulortman) [inline tests/collector/basic_pip/expected_output_data.json]: @davegaeddert This now catches and reports the recursive inclusion from your first pip collector test.

**Comment by paulortman** [comment]
I'm sorry about all the whitespace churn -- I don't know what, if anything caused it (I just did `dependencies test --update-output-data`).

**Comment by davegaeddert** [comment]
Are you running dependencies-cli under python2? I'm not positive, but at one point I thought that was the culprit: https://github.com/dependencies-io/cli/issues/62

File: tests/actor/dependencies_tests.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 cases:
-- name: Without pip freeze
+- name: Without_pip_freeze
```
Comment (davegaeddert) [inline tests/actor/dependencies_tests.yml]: Did you do this for `pytest -k`? I was thinking about requiring these be slugs for that reason...

Comment (paulortman) [reply tests/actor/dependencies_tests.yml]: Yup.  Even so, I'll probably rename all the tests so there is more uniqueness.  Trying '-k basic_pip' matches 'basic_pipfile' too...

**Pull request merged**